docker-compose/docker-compose.lb.selfsigned.yml

```diff
@@ -196,6 +196,25 @@ services:
     volumes:
       - kafka_data:/bitnami/kafka
 
+  init-kafka:
+    image: "docker.io/bitnami/kafka:${KAFKA_TAG}"
+    networks:
+      openwifi:
+    depends_on:
+      - kafka
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        TOPICS="command connection device_event_queue device telemetry healthcheck \
```

Comment:
Is it possible to move these variables into .env file in order to get rid of duplication in multiple files?

Reply:
makes sense, I've added the variable to kafka.env so it is shared by all deployments.